packages/react-ui-core/package.json

```diff
@@ -54,15 +55,16 @@
     "form-serialize": "0.7.2",
     "mapbox-gl": "0.42.2",
     "prop-types": "^15.6.1",
+    "react": "16.3.1",
```

Comment:
there a reason you added this here? I just had to fix a bug in hive related to this sort of thing. `dependencies` are really for private packages of the library. Since `react` is used by the app (rent-js/ag.js) and other packages, then all versions have to be in agreement. Which is what `peerDependencies` is for, for library packages.

Let's say rent-js installed 16.3.2, and this is locked to 16.3.1. Then you end up with:
```
node_modules/react  <-- 16.3.2
node_modules/@rentpath/react-ui-core/node_modules/react  <-- 16.3.1
```
Without aliases, webpack is going to bundle both of these versions which can cause JS errors.

(interesting note about this... `yarn link` also is problematic in similar manner if you `yarn install` inside a library package and then link that package to an app... npm, I swear 🤦‍♂️  )

Reply:
We could add react to the resolutions section, perhaps

```"resolutions": {
    "koa": "2.3.0",
    "react": "16.3.2",
}```

Reply:
Well, certainly not by using two versions of react at once 🤣 

The `peerDependencies` already has `"16.3.1"`, which should cause a warning on rent-js if that library version is not available (though it will be, since I put in a PR for it). I'm not sure why we locked those down (really, `"^16.3.1"` will give you versions >= 16.3.1 and < 17.0.0).

We really need to clean up peer dependencies on rent-js and start paying attention to the warnings. Letting the app  specify the version on react/react-dom and other "global" sort of packages is, IMO, the [right way to go](https://stackoverflow.com/a/34645112).

Reply:
We locked them down specifically for the reason this change addresses.  I see your point though, what do you think about adding `react-dom` as a peer as well?